tests/testthat/test-unnest.R

```diff
@@ -150,4 +150,7 @@ test_that("keep_empty", {
   expect_equal(out$vecs, c(NA, 1, 2))
 })
 
-
+test_that("handles empty tibble", {
+  expect_equal(unnest.(tibble::tibble(x = list()), x),
```

Comment:
Can you use `tidytable()` or `data.table()` here instead of `tibble::tibble()`? This won't pass CRAN checks because tidytable doesn't depend on tibble/dplyr. And can the title then be `"handles empty data frame"`.